src/sites/twitch-twilight/modules/chat/input.jsx

```diff
@@ -324,10 +324,18 @@ export default class Input extends Module {
 		inst.tempInput = '';
 		inst.messageHistoryPos = -1;
 
+		let element = this.fine.getChildNode(inst);
+		let container = element && element.querySelector('div[class="Layout-sc-nxg1ff-0 chat-input__input-icons"]');
+		container.append(<div class="Layout-sc-nxg1ff-0 appendChildForWrittenChars"><p id="textForCharsWritten">( 0 / 500 )</p></div>);
```

Comment:
Likewise, please avoid using CSS class names from `styled-components`. If you need to mimic the appearance of a Twitch element, we need to add our own CSS class(es) or re-use existing classes FFZ already adds.